examples/domainmodel/langium-config.json

```diff
@@ -5,5 +5,12 @@
     "out": "src/language-server/generated",
     "textMate": {
         "out": "syntaxes/domain-model.tmLanguage.json"
+    },
+    "parserConfig": {
+      "chevrotainConfig": { 
```

Comment:
I'd propose to simplify this structure by eliminating one nesting level and name the group `chevrotainParserConfig`.

Reply:
It's made specially for simple adding of other parser configurations. I'll remove it.